Userland/Libraries/LibGfx/Path.cpp

```diff
@@ -21,9 +21,9 @@ void Path::elliptical_arc_to(FloatPoint point, FloatSize radii, double x_axis_ro
     double rx = radii.width();
     double ry = radii.height();
 
-    double x_axis_rotation_c = AK::cos(x_axis_rotation);
-    double x_axis_rotation_s = AK::sin(x_axis_rotation);
-
+    double x_axis_rotation_s;
+    double x_axis_rotation_c;
+    AK::sincos(static_cast<double>(x_axis_rotation), x_axis_rotation_s, x_axis_rotation_c);
```

Comment:
Why the `static_cast<double>` here, it's already a double?

Reply:
Ah, mixed up the commits (I change it to a float in the next commit to be consistent with the other APIs which all take floats).